lib/grpcServer/handlers/core/getLastUserStateTransitionHashHandlerFactory.js

```diff
@@ -13,7 +13,25 @@ function getLastStateTransitionHashHandlerFactory(coreAPI) {
    * @param {function(Error, Object)} callback
    */
   function getLastStateTransitionHashHandler(call, callback) {
-    const userId = call.request.getUserId();
+    const { userId: userIdBuffer } = call.request;
+
+    if (!Buffer.isBuffer(userIdBuffer)) {
```

Comment:
It has to be a buffer otherwise protobuf will cause an error so we don't need to double check it.